ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/dynamic-dns/provision/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-dynamic-dns-provision-chain.yaml

```diff
@@ -0,0 +1,36 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-dynamic-dns-provision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-ovn-provision
+  - chain: cucushift-hypershift-extended-install-private
+  - chain: hypershift-aws-create
+  - ref: cucushift-hypershift-extended-enable-dns-update-record
+  - ref: cucushift-hypershift-extended-enable-dns-generate-certificate
+  - ref: cucushift-hypershift-extended-enable-dns-custom-kubeconfig
+  env:
+  - name: BASE_DOMAIN
+    default: "qe.devcluster.openshift.com"
+  - name: HYPERSHIFT_BASE_DOMAIN
+    default: "qe.devcluster.openshift.com"
+  - name: HYPERSHIFT_GUEST_INFRA_OCP_ACCOUNT
+    default: "true"
+  - name: HYPERSHIFT_AWS_REGION
+    default: ""
+  - name: HYPERSHIFT_HC_ZONES
+    default: ""
+  - name: HYPERSHIFT_CP_AVAILABILITY_POLICY
+    default: "HighlyAvailable"
+  - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
+    default: "HighlyAvailable"
+  - name: ZONES_COUNT
+    default: "3"
+  - name: ENABLE_ICSP
+    default: "true"
+  - name: COMPUTE_NODE_REPLICAS
+    default: "3"
+  - name: USAGE_CLUSTER_TYPE
+    default: "hypershift-mgmt"
+  - name: HYPERSHIFT_DYNAMIC_DNS
+    default: "hypershift-qe-dns.qe.devcluster.openshift.com"
```

Comment:
Is there any conflict when multiple jobs are running with the same dynamic dns?

Reply:
the dns only used in the profile, it's not a common dns used for other profile